ibis/expr/operations.py

```diff
@@ -720,6 +720,17 @@ class Quantile(Reduction):
     output_type = rules.type_of_arg(1)
```

Comment:
updated, though no test on the output (IIRC we are not actually checking via ibis anywhere, only as a test result).

> Also I think the number(name='quantile') should be double(name='quantile') 

we allow float & int input. (e.g. 0 and 1 are valid).